exchanges/gateio/gateio_test.go

```diff
@@ -2972,7 +2972,7 @@ func TestGenerateSubscriptionsSpot(t *testing.T) {
 	exp := subscription.List{}
 	for _, s := range g.Features.Subscriptions {
 		for _, a := range g.GetAssetTypes(true) {
-			if s.Asset != asset.All && s.Asset != a {
+			if s.Asset != asset.All && s.Asset != a || !g.IsAssetWebsocketSupported(a) {
```

Comment:
This is not pretty.
Make it pretty.

Reply:
Fixed https://github.com/thrasher-corp/gocryptotrader/pull/1693/commits/feed889b1e9f4b7124b20173bdd26fbd150f7a03